libs/designer-ui/src/lib/editor/base/nodes/tokenNode.tsx

```diff
@@ -66,6 +66,11 @@ export class TokenNode extends DecoratorNode<JSX.Element> {
     };
   }
 
+  // This is to enable copy to clipboard, even though there are some cases where @{} isn't needed, for the time being it's easier to always include it when copying
+  getTextContent(_includeInert?: boolean | undefined, _includeDirectionless?: false | undefined): string {
+    return `@{${this.__data.value}}` ?? '';
```

Comment:
any parameters as boolean, int, object or even non string types will have it stringified wrongly, but given they are copying it into notepad it might be ok [even if they complain about change in behavior]. But once you allow them to copy from clipboard to a different token content then that might break the runtime.

Reply:
I created a work item here to create a custom clipboard plugin to handle the full case #2681 , we can prioritize further if we get more complaints. I don't think this is that common of a workflow though so I'm ok not prioritizing this until we have to.